app/Console/Commands/ClearTrackCache.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * Pony.fm - A community for pony fan music.
+ * Copyright (C) 2015 Peter Deltchev
```

Comment:
Does this not belong to Poniverse, which is your IP? 

Just to confirm, should I add my name to files I've _created_, then?